src/contexts/AppContext.tsx

```diff
@@ -0,0 +1,209 @@
+import React, {createContext, useState, useContext, useEffect} from 'react';
+import AsyncStorage from '@react-native-community/async-storage';
+import {ResData, appService} from '../services/Service';
+import {FONT_SIZE} from '../constants/constants';
```

Comment:
Please rename `constants/constants.ts` to `constants/index.ts`, so it will be like just `constants` while importing.

Reply:
resolved